home-service/src/main/java/com/softserveinc/ita/homeproject/homeservice/service/cooperation/CooperationServiceImpl.java

```diff
@@ -57,7 +61,7 @@ public CooperationDto createCooperation(CooperationDto createCooperationDto) {
     private CooperationInvitationDto createInvitationForAdmin(CooperationDto cooperationDto) {
```

Comment:
Please change the arguments here so this method accepts cooperation id and admin email.